src/cli/cli.cpp

```diff
@@ -5744,6 +5746,85 @@ template <> otError Interpreter::Process<Cmd("radiofilter")>(Arg aArgs[])
 }
 #endif
 
+#if OPENTHREAD_CONFIG_RADIO_STATS_ENABLE
+inline unsigned long UsToSInt(uint64_t aUs) { return ToUlong(static_cast<uint32_t>(aUs / 1000000)); }
+inline unsigned long UsToSDec(uint64_t aUs) { return ToUlong(static_cast<uint32_t>(aUs % 1000000)); }
+
+void Interpreter::OutputRadioStatsTime(const char *aTimeName, uint64_t aTimeUs, uint64_t aTotalTimeUs)
+{
+    uint32_t timePercentInt = static_cast<uint32_t>(aTimeUs * 100 / aTotalTimeUs);
+    uint32_t timePercentDec = static_cast<uint32_t>((aTimeUs * 100 % aTotalTimeUs) * 100 / aTotalTimeUs);
+
+    OutputLine("%s Time: %lu.%06lus (%lu.%02lu%%)", aTimeName, UsToSInt(aTimeUs), UsToSDec(aTimeUs),
+               ToUlong(timePercentInt), ToUlong(timePercentDec));
+}
+
+template <> otError Interpreter::Process<Cmd("radio")>(Arg aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+
+    /**
+     * @cli radio stats
+     * @code
+     * radio stats
+     * Radio Statistics:
+     * Total Time: 67.756s
+     * Tx Time: 0.022944s (0.03%)
+     * Rx Time: 1.482353s (2.18%)
+     * Sleep Time: 66.251128s (97.77%)
+     * Disabled Time: 0.000080s (0.00%)
+     * Done
+     * @endcode
+     * #otRadioTimeStatsGet
+     */
+    if (aArgs[0] == "stats")
+    {
+        if (aArgs[1].IsEmpty())
+        {
+            const otRadioTimeStats *radioStats = nullptr;
+            uint64_t                totalTimeUs;
+
+            radioStats = otRadioTimeStatsGet(GetInstancePtr());
+
+            totalTimeUs =
+                radioStats->mSleepTime + radioStats->mTxTime + radioStats->mRxTime + radioStats->mDisabledTime;
+            if (totalTimeUs == 0)
+            {
+                OutputLine("Total Time is 0!");
+            }
+            else
+            {
+                OutputLine("Radio Statistics:");
+                OutputLine("Total Time: %lu.%03lus", ToUlong(static_cast<uint32_t>(totalTimeUs / 1000000)),
+                           ToUlong((totalTimeUs % 1000000) / 1000));
+                OutputRadioStatsTime("Tx", radioStats->mTxTime, totalTimeUs);
+                OutputRadioStatsTime("Rx", radioStats->mRxTime, totalTimeUs);
+                OutputRadioStatsTime("Sleep", radioStats->mSleepTime, totalTimeUs);
+                OutputRadioStatsTime("Disabled", radioStats->mDisabledTime, totalTimeUs);
+            }
+        }
+        /**
+         * @cli radio stats clear
+         * @code
+         * radio stats clear
+         * Done
+         * @endcode
+         * #otRadioTimeStatsReset
```

Comment:
Please add

`@par api_copy`

Before this line so that it imports the API comments into the CLI docs